python/ctsm/site_and_regional/mesh_type.py

```diff
@@ -0,0 +1,560 @@
+"""
```

Comment:
Since, this is a type that's not a top level program it should be moved to the python directory, so that the python building tools can be used around it.

Reply:
This is something that I must have misread the directory name. So resolving.